docs/docs/your-fable-project/author-a-fable-library.md

```diff
@@ -18,7 +20,89 @@ The last point may sound complicated but it's only a matter of adding a couple o
 </ItemGroup>
 ```
 
-That's all it takes to make your library compatible with Fable! In order to publish the package to Nuget check [the Microsoft documentation](https://docs.microsoft.com/en-us/nuget/quickstart/create-and-publish-a-package-using-the-dotnet-cli) or alternatively you can also [use Fake](https://fake.build/dotnet-nuget.html#Creating-NuGet-packages).
+In order to publish the package to Nuget check [the Microsoft documentation](https://docs.microsoft.com/en-us/nuget/quickstart/create-and-publish-a-package-using-the-dotnet-cli) or alternatively you can also [use Fake](https://fake.build/dotnet-nuget.html#Creating-NuGet-packages).
+
+## Make your package discoverable
+
+[Fable.Packages](https://fable.io/packages/) is a tool making it easy for users to search for Fable packages. 
+
+To make your packages listed on Fable.Packages, you need to add some tags to your `.fsproj`.
+
+<ul class="textual-steps">
+
+<li>
+
+**All Fable** packages must have the `fable` tag.
+
+</li>
+
+<li>
+
+Specify what kind of package you are publishing.
+
+A fable package can be one of the following:
+
+- `fable-library`: A library that can be used in Fable
+
+    Examples of libraries could be [Fable.Promise](https://github.com/fable-compiler/fable-promise/), [Elmish](https://elmish.github.io/), [Thoth.Json](https://thoth-org.github.io//Thoth.Json/), [Feliz](https://zaid-ajaj.github.io/Feliz/)
+
+    <div></div> <!-- Force a space to improve visual -->
+
+- `fable-binding`: The package consist of a set of API to make a native library available
+
+    For example:
+    
+    - A package which makes an NPM package API available
+    - A package which makes the Browser API available
+    - A package which makes a cargo package API available
+
+</li>
+
+<li>
+
+Specify which targets are supported by your package.
+
+Choose one or more of the following tags:
+
+- `fable-dart`: Dart is supported by the package 
+- `fable-dotnet`: .NET is supported by the package 
+- `fable-javascript`: JavaScript is supported by the package 
+- `fable-python`: Python is supported by the package 
+- `fable-rust`: Rust is supported by the package 
+- `fable-all`: Package is compatible with all Fable targets.
+
+    :::warning
+    A package can only be compatible with all the target if it depends only packages that are also compatible with all the targets.
+
+    A package compatible with all the targets cannot be a binding as bindings are target specific.
```

Comment:
Rephrasing a bit:

    A package can be compatible with all targets if it depends only on packages that are also compatible with all targets.

    A package compatible with all targets cannot be a binding, as these are target-specific.